src/generate/CodeModelAzImpl.ts

```diff
@@ -1009,6 +1020,9 @@ export class CodeModelCliImpl implements CodeModelAz {
 
     public get CommandGroup_Mode(): string {
         if (isNullOrUndefined(this.CommandGroup?.language?.['cli']?.extensionMode)) {
+            if (this.IsCliCore && this.Extension_Mode === ExtensionMode.Stable) {
```

Comment:
why is the logic here?

Reply:
generally, the command group layer mode will take the global extension mode value if there's no command group layer mode specified. 
For example if the global extension mode is set as experimental we don't need to set each command group as experimental again. That's why we have the previous changes here https://github.com/Azure/autorest.az/pull/797/files#diff-2c3dfede02f6d90578baaa72578f79e8572b6cb71c5449f3237b5adfe2f943c4L63-L162 
but issue #763 says that we shouldn't label global extension mode as experimental when generating main repo modules. instead we should put the experimental label down to each generated command group layer.